modules/govuk_jenkins/templates/jobs/deploy_app_downstream.yaml.erb

```diff
@@ -9,15 +9,6 @@
           days-to-keep: 30
           artifact-num-to-keep: 5
     builders:
-      <% if @smokey_pre_check %>
-      - shell: |
-          # Wait for any app restarts to complete before running Smokey
-          sleep 60
-      - trigger-builds:
-          - project: Smokey
-            current-parameters: true
-            block: true
-      <% end %>
       - shell: |
           # Workaround for our inconsistent repo vs. deployment naming
           case "$TARGET_APPLICATION" in
```

Comment:
I think this bit needs to stay where it was. Fortunately, it's only used there. It's probably worth doing a test deployment of this on Integration, just in case it doesn't work.

Reply:
`$TARGET_APPLICATION` is needed for the call to the release API too. I assumed environment variables are rolling over to subsequent shell scripts, because it's used again in line 65.

Reply:
But I'll test in integration and write back with the results!

Reply:
> $TARGET_APPLICATION is needed for the call to the release API too.

Yup. I'm referring to the whole case statement, which sets a `REPO` variable that's only needed in the final step.

Reply:
Gottcha - soo my brain was thinking this was actually setting the TARGET_APPLICATION variable 🤦  - I'll shift it down